docker-compose-config.py

```diff
@@ -76,4 +79,7 @@ def main():
 
 
 if __name__ == "__main__":
-    main()
+    try:
+        main()
+    except KeyboardInterrupt:
```

Comment:
I don't get this - why do we need to handle this separately?

If user pushes ctrl+c it's 'normal' for python to exit with a stack trace, as do other languages.

Reply:
It's just for a cleaner exit, the stack trace is rather irrelevant in the case of a script like this

Reply:
It seems boilerplatey for a script but leaving it up to you.